frontend/src/metabase/static-viz/index.js

```diff
@@ -67,3 +70,17 @@ export function RenderChart(rawSeries, dashcardSettings, options) {
     />,
   );
 }
+
+export function RenderMinibar(data) {
```

Comment:
Is there any noticeable impact of this on performance? I assume there can be some interop cost but maybe it is negligible